app/controllers/admin/reports_controller.rb

```diff
@@ -80,4 +80,28 @@ def committee_member_report_export
       headers['Content-Type'] ||= 'text/xls'
     end
   end
+
+  def graduate_data_report_export
+    @report_export = ExportCsv.new('graduate_data_report', graduate_data_result)
+
+    respond_to do |format|
```

Comment:
This is new to me, but seems really handy! Nice to have this shortcut for returning json in just a few lines.